vision/api/VisionTest/VisionTest.cs

```diff
@@ -195,6 +197,25 @@ public void DetectCropHint()
             Assert.Contains("\tX:", output.Stdout);
             Assert.Contains("\tY:", output.Stdout);
         }
+
+        [Fact]
+        public void DetectDocument()
+        {
+            var bucketName = System.Environment.GetEnvironmentVariable("CLOUD_STORAGE_BUCKET");
+            var outputPrefix = "OCR_PDF_TEST_OUTPUT";
+            var gcsSourceURI = $"gs://{bucketName}/HodgeConj.pdf";

```

Comment:
Please put the .pdf being used to test into `./VisionTest/data/`

Also, if it is not already, please use the same pdf file as the Python tests use

Finally, my personal CLOUD_STORAGE_BUCKET does not contain this file so this will presumably fail. Please use the Storage helpers to upload this file before the tests, presuming it's not already

See: https://github.com/telpirion/dotnet-docs-samples/blob/2dd06fa770c4688370ae62c5f7de9cedf83b5fa3/vision/api/VisionTest/VisionTest.cs#L242-L266

Reply:
<img width="663" alt="screen shot 2018-07-16 at 12 13 28 pm" src="https://user-images.githubusercontent.com/280/42778571-6444c702-88f2-11e8-84b1-b40ae48e7ebb.png">

Reply:
Annoyingly, this is not where it should be in the python-docs-samples (we should fix that)

https://github.com/GoogleCloudPlatform/python-docs-samples/tree/master/vision/cloud-client/detect/resources